mautic/mautic
# TPROD-314 - DynamicContentBundle - Migrate PHP Templates to Twig Templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. 



- `/s/dwc`
  - page renders with and without dynamic content
  - filter
  - sort columns
  - navigate multiple pages
- `/s/dwc/new` and `/s/dwc/edit/{id}`
  - page renders correctly
  - filters render correctly
  - **NOTE** see known issues
- `/s/dwc/view/{id}`
  - page renders without errors
  - translations tab only appears if the dynamic content is a translation of or has translations

https://github.com/mautic/mautic/pull/11797

**Known Issues**
* https://github.com/mautic/mautic/issues/11730
* https://github.com/mautic/mautic/issues/11290
* When creating or editing dynamic content, when you select **Create new** from the drop down on the **Is a translation of** field, it will not create a new dynamic content. It just updates the existing dynamic content and the modal stays open.
* Cloning an existing dynamic content will not clone the filters
* If you find a bug with the functionality, check the 4.x or 5.x branches to verify it exists there, if it does please let me know so I can see if it's an easy fix or not
